AUTHORS

```diff
@@ -4,11 +4,14 @@ Developers:
 Nick Treleaven <nick(dot)treleaven(at)btinternet(dot)com>
 Colomban Wendling <lists(dot)ban(at)herbesfolles(dot)org>
```

Comment:
probably yeah, as I don't use that email address anywhere else for Geany (in a visible fashion anyway)